src/chatlog/chatlog.cpp

```diff
@@ -830,3 +828,13 @@ void ChatLog::focusOutEvent(QFocusEvent* ev)
             lines[i]->selectionFocusChanged(false);
     }
 }
+
+void ChatLog::keyPressEvent(QKeyEvent * event)
+{
+    if(event->matches(QKeySequence::Copy))
```

Comment:
Put a space between the `if` and the paren, as per the newly-refurbished code style guide on the wiki :)